README.rst

```diff
@@ -24,6 +24,14 @@ Platforms support
 
 Python-inquirer supports mainly UNIX-based platforms (eq. Mac OS, Linux, etc.). Windows has experimental support, please let us know if there are any problems!
 
+Installation
+============
+
+.. code:: bash
```

Comment:
Please change to:
.. code:: console

So that syntax highlighting works better.